### PR TITLE
The calculator UI was created

### PR DESCRIPTION
**Pull request by jnicolaslaurenti**
The new calculator, now with a face wash

Pixel 3A:

image

Nexus S:

image

### CODE REVIEW

**Review by SartiMau — APPROVED** [review]
Check the last evidence. The image is overlapped by the buttons

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -4,15 +4,224 @@
...
+        android:text="@string/activity_main_calculator_multiply_text"
+        app:layout_constraintBottom_toTopOf="@id/button_subtraction"
+        app:layout_constraintEnd_toEndOf="@id/button_subtraction"
+        app:layout_constraintStart_toStartOf="@id/button_subtraction" />
+
+    <Button
+        android:id="@+id/button_one"
+        style="@style/button_style"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
```
Comment (mauroLonghi) [inline app/src/main/res/layout/activity_main.xml]: this is not right, try with that :
android:layout_width="0dp"
android:layout_height="0dp"
app:layout_constraintWidth_default="wrap"
app:layout_constraintHeight_default="wrap"
this set as wrap all this dimensions

**Review by SartiMau — CHANGES_REQUESTED** [review]
Fix the landscape UI pls

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (jnicolaslaurenti) [reply app/src/main/res/layout/activity_main.xml]: Solved!

File: app/src/main/java/com/onboarding/calculatorkotlin/MainActivity.kt [hunk 2]
```diff
@@ -1,10 +1,12 @@
 package com.onboarding.calculatorkotlin
 
+import android.content.pm.ActivityInfo
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
+        setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
```
Comment (BQuinteros) [inline app/src/main/java/com/onboarding/calculatorkotlin/MainActivity.kt]: this is not the correct form to portrait the activity. Its more easily if you put it in the manifest

**Review by lucianoeli — APPROVED** [review]

**Review by BQuinteros — APPROVED** [review]

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -4,15 +4,232 @@
...
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@id/button_clean"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <Button
+        android:id="@+id/button_clean"
+        style="@style/button_style"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_between_button_xsmall"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: as you use these margins in several buttons, please use a style

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -4,15 +4,232 @@
...
         app:layout_constraintTop_toTopOf="parent" />
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+    <TextView
+        android:id="@+id/text_view_result"
+        style="@style/result_style"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_marginBottom="@dimen/margin_bottom_result_xbig"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: you can put this property into the result_style

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -4,15 +4,240 @@
...
+        app:layout_constraintEnd_toStartOf="@id/button_clean"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <Button
+        android:id="@+id/button_clean"
+        style="@style/button_style"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        app:layout_constraintHeight_default="wrap"
+        android:layout_marginHorizontal="@dimen/margin_between_button_xsmall"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: you use this margin in several buttons, please create a new style for this if it is possible

File: app/src/main/res/layout/activity_main.xml (hunk 4 above)
Comment (SartiMau) [reply app/src/main/res/layout/activity_main.xml]: this still here

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/onboarding/calculatorkotlin/MainActivity.kt (hunk 2 above)
Comment (jnicolaslaurenti) [reply app/src/main/java/com/onboarding/calculatorkotlin/MainActivity.kt]: Solved!

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (jnicolaslaurenti) [reply app/src/main/res/layout/activity_main.xml]: Solved!

File: app/src/main/res/layout/activity_main.xml (hunk 4 above)
Comment (jnicolaslaurenti) [reply app/src/main/res/layout/activity_main.xml]: Solved!

File: app/src/main/res/layout/activity_main.xml (hunk 5 above)
Comment (jnicolaslaurenti) [reply app/src/main/res/layout/activity_main.xml]: Done!

**Review by BQuinteros — APPROVED** [review]

**Review by nicolbidone — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

**Review by SartiMau — APPROVED** [review]

**Pull request merged**